### DISCUSSION TITLE
How to build a config with REALITY + ws + h2 + gRPC + TCP  + naiveProxy + ss ？

### DISCUSSION THREAD

**Discussion by cary-sas** [lxhao61/integrated-examples#160] [General]
Hello,
I want to update my current config setting from " **XTLS** + ws + h2 + gRPC + TCP  + naiveProxy + ss " to " **REALITY** + ws + h2 + gRPC + TCP  + naiveProxy + ss ",  would you please guide me on how to achieve this?  see my current config in here,  [https://github.com/cary-sas/across/tree/master/xray/etc](https://github.com/cary-sas/across/tree/master/xray/etc),

**Comment by lxhao61** [comment]
参考示例：[Xray(M+B+D+G+A)+Caddy(N+T)](https://github.com/lxhao61/integrated-examples/tree/main/Xray(M%2BB%2BD%2BG%2BA)%2BCaddy(N%2BT))

**Comment by cary-sas** [comment]
Yes, I have seen this example. However, there is one uncertain part below that I'm unsure how to handle. The parameter **"dest": "@https"** needs to be connected to my Caddy configuration file. It's quite challenging for me to complete this task. Would you mind taking a look at my current configuration file? You can find it at [https://github.com/cary-sas/across/tree/master/xray/etc](https://github.com/cary-sas/across/tree/master/xray/etc).

**Comment by lxhao61** [comment]
“@http2.sock"是UDS连接时采用Abstract sockets模式，其介绍可参考 fallbacks 配置中["dest"](https://xtls.github.io/config/features/fallback.html#fallbackobject)说明。

**Comment by cary-sas** [comment]
Sorry, I'm still having difficulty with this particular section,  for this place  ` "fallbacks": [{  "dest": "@vlessh2c.sock"`,  I can update by myself, but for   `"realitySettings": {  "show": false, "dest": "@http2.sock",`  I still don't know how to deal with it in my case.

**Comment by lxhao61** [comment]
晕！！
Xray的`"dest": "@http2.sock",`对应Caddy的`"listen": ["unix/@http2.sock"],`。

**Comment by cary-sas** [comment]
Sorry for bothering you again,  but you see my current Caddy config, would you please tell me how to update it accordingly?

[https://github.com/cary-sas/across/blob/master/xray/etc/caddy.json](https://github.com/cary-sas/across/blob/master/xray/etc/caddy.json)

**Comment by lxhao61** [comment]
以我的Caddy配置为准即可！根本无必要以你的Caddy配置来更新，不精简、冗余（嵌套）。
